Src/VsVimShared/VsVimHost.cs

```diff
@@ -607,13 +607,13 @@ private bool MoveFocusHorizontally(int indexDelta)
             topLevelWindows.Sort((a, b) => a.Left < b.Left ? -1 : 1);
             var indexOfActiveDoc = topLevelWindows.FindIndex(win => win == _dte.ActiveWindow);
             var movedIndex = indexOfActiveDoc + indexDelta;
-            var newIndex = (movedIndex < 0 ? movedIndex + topLevelWindows.Count : movedIndex % topLevelWindows.Count);
+            var newIndex = (movedIndex < 0 ? movedIndex + topLevelWindows.Count : movedIndex % (double)topLevelWindows.Count);

```

Comment:
Rather than casting to `double` here I think we should just catch the case where count is `0` and handle that specifically. Likely by just returning false.